src/module.ts

```diff
@@ -116,7 +116,9 @@ export default defineNuxtModule<ModuleOptions>({
       return
     }
     // couldn't be found for some reason, assume compatibility
-    const { version: unheadVersion } = await readPackageJSON('@unhead/vue')
+    const { version: unheadVersion } = await readPackageJSON('@unhead/vue', {
+      from: import.meta.url,
```

Comment:
maybe use nuxt.options.modulesDir?

there are some examples in nuxt repo

Reply:
👍 thanks !